registrator.go

```diff
@@ -172,6 +173,10 @@ func main() {
 			go b.Add(msg.ID)
 		case "die":
 			go b.RemoveOnExit(msg.ID)
+		case "kill":
+			if *deregisterOnStop {
+				go b.RemoveOnExit(msg.ID)
+			}
 		}
```

Comment:
Hi @mattaitchison. Do you have any additional feedback for this PR, or any replies to my comments above?